libr/bin/format/objc/mach0_classes.h

```diff
@@ -0,0 +1,25 @@
+#include <r_bin.h>
+
+#undef Mach0_class_
+#undef Mach0_struct_
+#undef mach0_ut
+#undef r_bin_plugin_mach
+
+#ifdef R_MACH0_CLASS64
+#define Mach0_class_(name) Mach0_class64_##name
```

Comment:
I think we don't need both Mach0_class_ and Mach0_struct_. We may do like ELF with only Mach0_.
It's just enough to differentiate between 64 and 32 mach0, not between methods and structure.